common/model-views.cpp

```diff
@@ -3277,12 +3234,29 @@ namespace rs2
         for (auto&& n : related_notifications) n->dismiss(false);
     }
 
+
     void device_model::refresh_notifications(viewer_model& viewer)
     {
         for (auto&& n : related_notifications) n->dismiss(false);
 
         auto name = get_device_name(dev);
 
+        try {
+            dev_updates_profile profile(dev, "http://realsense-hw-public.s3-eu-west-1.amazonaws.com/rs-tests/sw-update/21_05_2020/rs_versions_db.json");
```

Comment:
PLs refactor the link to constexpr in the beginning of the file. Maintainability

Reply:
Agreed

Reply:
Done

Reply:
Done